### DISCUSSION TITLE
When I have so many routers, how to config it.

### DISCUSSION THREAD

**Discussion by NOVA-ME** [tokio-rs/axum#246] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/18442141/130458270-fa9142ae-ed79-4869-b20e-c74ab806b511.png)
I think write them all in main is too bad. But I can't find other method to do.

**Comment by davidpdrsn** [comment, marked as answer]
You can this https://docs.rs/axum/0.1.3/axum/#nesting-applications
